shared/devices/device-revoke/index.js

```diff
@@ -1,31 +1,49 @@
 // @flow
 import * as React from 'react'
-import {Confirm, Box, Text, Icon, type IconType} from '../../common-adapters'
-import {globalColors, globalMargins, globalStyles} from '../../styles'
+import {type DeviceType} from '../../constants/types/devices'
+import {Confirm, Box, Text, Icon, ProgressIndicator, type IconType} from '../../common-adapters'
+import {globalColors, globalMargins, globalStyles, styleSheetCreate, isMobile} from '../../styles'
 
 export type Props = {
   currentDevice: boolean,
   deviceID: string,
   endangeredTLFs: Array<string>,
-  icon: IconType,
+  type: DeviceType,
   name: string,
   onCancel: () => void,
   onSubmit: () => void,
   waiting: boolean,
 }
 
-const Header = ({name, icon}) => (
-  <Box style={{...globalStyles.flexBoxColumn, alignItems: 'center'}}>
-    <Icon type={icon} />
-    <Text type="BodySemibold" style={styleName}>
-      {name}
-    </Text>
-  </Box>
-)
+const Header = ({name, type}: {name: string, type: DeviceType}) => {
+  const icon: IconType = {
+    backup: isMobile ? 'icon-paper-key-revoke-64' : 'icon-paper-key-revoke-48',
+    desktop: isMobile ? 'icon-computer-revoke-64' : 'icon-computer-revoke-48',
+    mobile: isMobile ? 'icon-phone-revoke-64' : 'icon-phone-revoke-48',
+  }[type]
+  return (
+    <Box style={{...globalStyles.flexBoxColumn, alignItems: 'center'}}>
```

Comment:
should move to styles

Reply:
Generally, are there any cases where styles should be defined inline like this or are we moving to the practice of putting all styles in the `styles` object?

Reply:
ideally they'd all be in there. plus you already have a styleSheetCreate so might as well put it in